Makefile

```diff
@@ -57,188 +56,196 @@ ifeq ($(OPTIMIZE), true)
 endif
 	#
 
-$(OUT)/trivialDAO_storage.tz : admin_address = tz1QozfhaUW4wLnohDo6yiBUmh7cPCSXE9Af
-$(OUT)/trivialDAO_storage.tz : guardian_address = KT1QbdJ7M7uAQZwLpvzerUyk7LYkJWDL7eDh
-$(OUT)/trivialDAO_storage.tz : governance_token_address = KT1RdwP8XJPjFyGoUsXFQnQo1yNm6gUqVdp5
-$(OUT)/trivialDAO_storage.tz : governance_token_id = 0n
-$(OUT)/trivialDAO_storage.tz : now_val = `date +"%s"`
-$(OUT)/trivialDAO_storage.tz : metadata_map = (Big_map.empty : metadata_map)
-$(OUT)/trivialDAO_storage.tz : fixed_proposal_fee_in_token = 0n
-$(OUT)/trivialDAO_storage.tz : ledger = ([] : ledger_list)
-$(OUT)/trivialDAO_storage.tz : quorum_threshold = 10n
-$(OUT)/trivialDAO_storage.tz : min_quorum = 1n
-$(OUT)/trivialDAO_storage.tz : max_quorum = 99n
-$(OUT)/trivialDAO_storage.tz : period = 950400n # 11 days
-$(OUT)/trivialDAO_storage.tz : quorum_change = 5n
-$(OUT)/trivialDAO_storage.tz : max_quorum_change = 19n
-$(OUT)/trivialDAO_storage.tz : governance_total_supply = 100n
-$(OUT)/trivialDAO_storage.tz : proposal_flush_time = 1900801n # 22 days 1 seconds
-$(OUT)/trivialDAO_storage.tz : proposal_expired_time = 2851200n # 33 days
```

Comment:
Oh, you ended up making them all required, I was thinking of removing the default value only for some of them.

See the issue description again, I think it's pretty clear that what fields to require should be decided on first, especially from the first acceptance criteria:
> which of the Makefile storage targets argument should always be provided by the user is established and default values are no longer provided for these

I don't think they all fit the condition that they "should be provided" by the user.

I mean, my point is that with the default e.g. `admin_address` or `guardian_address` a real contract won't actually make sense, so the user should provide them in any case.
However, values like the one of `period` or `quorum_change` have decent default values which can be used for a real deployment, so we can keep giving defaults that can be overridden.

Reply:
No problem, perhaps I should have made it more clear.

Yes, I think those should be all the required values :+1:

Reply:
If that's the case, I've pushed the update. You can take a look at it.